articles/quickstart/spa/auth0-react/01.md

```diff
@@ -0,0 +1,299 @@
+      ) : (
+        <>
+          <NavBar />
+          <Switch>
+            <Route path="/" exact component={Home} />
+            <PrivateRoute path="/profile" component={Profile} />
+          </Switch>
+        </>
+      )}
+    </div>
+  );
+}
+```
+
+⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️
+
+**TODO: Final URL to sample.**
+ 
+⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️
+
+Take a look at this sample that puts this technique together while also implementing a navigation bar: "Using React Router with the Auth0 React SDK" [sample required]
+
+## Call an API
+
+Your React application may need to access protected data from an API that you are also [securing with Auth0](https://auth0.com/docs/microsites/protect-api/protect-api). For that use case, you'll need to retrieve an access token in the format of a [JSON Web Token (JWT)](https://jwt.io/) to authorize any calls you make to that secured API. 
+
+For the Auth0 React SDK to configure the access token correctly, you need to pass an `audience` property to the `Auth0Provider` as follows:
+
+```javascript
+import React from "react";
+import ReactDOM from "react-dom";
+import App from "./App";
+
+import { Auth0Provider } from "@auth0/auth0-react";
+
+ReactDOM.render(
+  <Auth0Provider
+    domain="${account.namespace}"
+    client_id="${account.clientId}"
+    audience="${apiIdentifier}"
+    redirect_uri={window.location.origin}
+  >
+    <App />
+  </Auth0Provider>,
+  document.getElementById("root")
+);
+```
+
+The set up is similar to the one discussed in the "Configure the SDK" section: you wrap your root component with `Auth0Provider` and use the "Settings" values of the single-page application you've registered with Auth0 to populate the values of `domain` and `client_id`. 
+
+However, this time you also pass it an `audience` prop to allow the API to validate the access token properly. When you [set up an API](https://auth0.com/docs/getting-started/set-up-api) with Auth0, you define its audience value, which you can find as the "Identifier" value under the API "Settings."
+
+Once you configure `Auth0Provider`, you can easily get the access token using the `getToken()` method from the `useAuth0()` custom React Hook wherever you need it. Take this `Account` component as an example:
+
+```javascript
+import React, { useEffect, useState } from "react";
+import { useAuth0 } from "@auth0/auth0-react";
+
+const Account = () => {
+  const { getToken } = useAuth0();
+  const [data, setData] = useState(null);
+
+  useEffect(() => {
+    (async () => {
+      const token = await getToken();
+
+      const response = await fetch("http://localhost:7000/protected-hello", {
+        headers: {
+          Authorization: `Bearer <%= "${token}" %>`,
+        },
+      });
+      const json = await response.json();
+
+      setData(json);
+    })();
+  }, [getToken]);
+
+  return (
+    <div>{data ? <span>{data.message}</span> : <span>Loading...</span>}</div>
```

Comment:
Error handling is probably too much for the quickstart - but I'd add a `loading` state in here

Reply:
Moved to CF.